articles/ros2_access_control_policies.md

```diff
@@ -0,0 +1,317 @@
+---
+layout: default
+title: ROS 2 Access Control Policies
+permalink: articles/ros2_access_control_policies.html
+abstract:
+  This article specifies the policy format used for access control when securing ROS subsystem.
+author:  >
+  [Ruffin White](https://github.com/ruffsl),
+  [Kyle Fazzari](https://github.com/kyrofa)
+
+published: true
+categories: Security
+---
+
+{:toc}
+
+
+# {{ page.title }}
+
+<div class="abstract" markdown="1">
+{{ page.abstract }}
+</div>
+
+Original Author: {{ page.author }}
+
+
+[SROS 2](/articles/ros2_dds_security.html) introduces several security properties, including encryption, authentication, and authorization, where authorization is subsequently obtained by combining the first two properties with a model for access control.
+Such models are often referred to as access control policies.
+A policy serves as a high-level abstraction of privileges associated with attributes that may then be transpiled into low-level permissions for individual identities, such as specific ROS nodes within a secure DDS network.
+
+## Concepts
+
+Before detailing the SROS 2 policy design of the [access control](https://en.wikipedia.org/wiki/Computer_access_control), by which the system constrains the ability of a subject to access an object, it is important to establish a few concepts useful in formalizing the design approach in terms of security.
+In this context, a subject may be thought of as a participant on a distributed data-bus (e.g. a ROS node in the computation graph), whereas an object may be an instance of a particular subsystem (e.g. a ROS topic), and access is defined as the capability to act upon that object (e.g. publish or subscribe).
+
+### Mandatory Access Control
+
+[Mandatory Access Control](https://en.wikipedia.org/wiki/Mandatory_access_control) (MAC) refers to allowing access to an object if and only if rules exist that allow a given subject to access the resource; the term mandatory denotes this requirement that a subject’s access to an object must always be explicitly provisioned.
```

Comment:
It would be good to emphasize on MAC are controls that cannot be bypassed.
```suggestion
Most importantly, contrary to discretionary access control (DAC), such policies are enforced by a set of authorization rules that cannot be overridden or modified by the subject either accidentally or intentionally.
```

Reply:
A fair distinction.